docs/UserGuide.adoc

```diff
@@ -12,134 +12,340 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3
+:repoURL: https://github.com/AY1920S1-CS2103T-T13-1/main
+:javaURL: https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `CS2013T-T13-1`      Since: `Sep 2019`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 3 (AB3) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB3 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+ORGANice Transplant Manager (OTM) is a desktop application for hospital administration staff +
+to manage organ transplants.
+With OTM, you can easily do the following:
+
+* organise information about patients, doctors, and donors
+* find matches between organ donors and patients
+
+
+OTM is *optimized for those who prefer to work with a Command Line Interface* (CLI) +
+while still having the benefits of a Graphical User Interface (GUI). If you can type fast, +
+OTM can get your administration tasks done faster than traditional GUI apps. Interested? +
+Jump to the <<Quick Start>> to get started. Enjoy!
 
 == Quick Start
 
-.  Ensure you have Java `11` or above installed in your Computer.
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
-.  Double-click the file to start the app. The GUI should appear in a few seconds.
+Below is a list of things to do to use the application:
+
+.  Ensure you have Java `11` or above installed in your computer. +
+If not, install it link:{javaURL}[here].
+.  Download the latest `organice.jar` link:{repoURL}/releases[here].
+.  Copy the jar file to the folder you want to use as the home folder for your application.
+.  Double-click the file to start the application. The application should appear in a few seconds. The following screenshot +
```

Comment:
not `double-click` i think use `java -jar organice.jar`

Reply:
Hi William, this is an instruction to go to the releases to download the jar fie. 
We can double click jar files to open them